telebot/plugins/notification_to_user.py

```diff
@@ -0,0 +1,39 @@
+"""
+This plugin is to notification to user who is tagged. And this plugin need to
+have a configuration yaml file as template:
+
+For example:
+[
+    {"name": "@namnh307", "id": 473649555},
+    {"name": "@daidv", "id": 432649532}
+]
+"""
+
+import logging
+import json
+
+LOG = logging.getLogger(__name__)
+
+
+def check_messsage(list_user, message):
```

Comment:
s/list_user/users/g

Reply:
Btw, notification_to_user is too long for plugin's name. You always complain about "stackalytics" name but now, you use a much longer name....

Reply:
notification is a noun, we should use a verb for plugin name. Just saying.